python/ray/_private/worker.py

```diff
@@ -2719,7 +2719,6 @@ def remote(
     resources: Dict[str, float] = Undefined,
     accelerator_type: str = Undefined,
     memory: Union[int, float] = Undefined,
-    object_store_memory: int = Undefined,
```

Comment:
Is if safe to just remove this param btw?

Reply:
Yes, it's just for type annotation.